imports/core/ui/containers/IdentifiedRoute.jsx

```diff
@@ -3,8 +3,20 @@ import { withTracker } from "meteor/react-meteor-data";
 import React from "react";
 
 import { Players } from "../../api/players/players.js";
+import { playerWasArchived } from "../../api/players/methods.js";
 
 class IdentifiedRouteInner extends React.Component {
+  componentDidMount() {
```

Comment:
When the app loads, it check if the "remembered" player ID was cleared. It does require the user to reload the app, it is not dynamic, but since they'd be coming back the next days or something like that, it shouldn't be a problem I think.